generators/readme/test/fixtures/avenger/src/avenger.js

```diff
@@ -0,0 +1,77 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const Avenger = (props) => <div>{`Avenger: ${props.name}`}</div>;
+Avenger.defaultProps = {
```

Comment:
can we add a default value for a prop boolean type? Just to have most of the cases covered

Reply:
Sure I can do that.

PS: we can also increase the "test coverage" as we migrate packages and bump into "unsupported" use cases.